### ISSUE TITLE
Phone number fields with country dropdown not working properly in IE11/Edge

### ISSUE THREAD

**Issue by jwrudzin** [verbb/formie#318]
**Description**
Phone number fields with country dropdowns are not working properly in IE11 or Edge. The country dropdown does not appear.

**Steps to reproduce**
1. Create a form with a phone number field with country dropdown.
2. Navigate to the form in IE11 or Edge, the field will not display properly.

### Form settings
- Multi-page form: No
- Submission Method: Page Reload
- Client-side Validation: Yes
- Custom Form Templates: No

**Additional info**
- Plugin version: 1.3.13
- Craft version: 3.5.18
- Multi-site: No

**Comment by engram-design** [comment]
Should be fixed in the next release. To get the fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.3.13",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.3.14](https://github.com/verbb/formie/releases/tag/1.3.14)
